elements/pfe-styles/demo/index.html

```diff
@@ -2,13 +2,13 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Layouts Demo</title>
-
-    <link rel="stylesheet" type="text/css" href="//overpass-30e2.kxcdn.com/overpass.css">
-    <link rel="stylesheet" type="text/css" href="../dist/pfe-base.min.css">
-    <link rel="stylesheet" type="text/css" href="../dist/pfe-layouts.min.css">
+    <title>PFE styles demo</title> 
```

Comment:
```suggestion
    <title>Layouts demo</title> 
```
I think this original title was clearer, especially since you separated out the other style aspects into separate demo pages.

Reply:
ah yeah, I renamed it before splitting it 👍